docs/docs-nav.yml

```diff
@@ -12,6 +12,8 @@
         url: /docs/default-locale-timezone/
       - title: "Disable Based on DisplayName"
```

Comment:
I like that you refer to parameterized tests, maybe adapt the title here as well (i.e. "Disable Parameterized Test Based on Display Name").